docs/src/main/asciidoc/_configprops.adoc

```diff
@@ -0,0 +1,110 @@
+|===
+|Name | Default | Description
+
+|spring.cloud.gcp.bigquery.credentials.default-scopes |  | 
+|spring.cloud.gcp.bigquery.enabled | true | Auto-configure Google Cloud BigQuery components.
```

Comment:
Does this mean that such a file is generated from the source code?
And whenever we change the code this one is going to be regenerated.

My point is that, please, pay attention that `default-scopes` goes without description and I believe that has to be fixed.

I don't say that you have to do this in this PR: just want to be sure what is needed to be done in the future.

Thanks

Reply:
Yeah this PR is just to enable the auto generation of the file and adding a link to the docs for it.  If there are issues with the specific properties we should fix them separately